src/cli/project.py

```diff
@@ -8,13 +8,85 @@
 import src.cli.console as console
 from src import settings
 from src.graphql import GraphQL
-from src.helpers import check_running_cluster, select_entity, select_entity_from_cluster_list
+from src.helpers import (
+    check_running_cluster,
+    get_list_of_project_ids_by_organization_id,
+    get_organization_id_by_title,
+    get_projects_by_organization_id,
+    select_entity,
+    select_entity_from_cluster_list,
+)
 from src.keycloak.permissions import KeycloakPermissions
 from src.local.providers.types import K8sProviderType
 from src.local.system import Telepresence
 from src.storage.user import get_local_storage_user
 
 
+def get_projects_for_organization(ctx, organization):
```

Comment:
Maybe we can move all "helper" functions with are not direct CLI commands into seperate helper files?
Currently we have some here and some in "src.helpers".

Reply:
That's good idea, as it would keep code for command groups cleaner. Maybe we can also unload complexity of several other commands to those helper functions.